packages/fints/src/parse.ts

```diff
@@ -37,4 +39,24 @@ export const Parse = {
         while (str.startsWith("0")) { str = str.substr(1, str.length); }
         return Number(str);
     },
+    /**
+     * Parse a date.
+     *
+     * @param str The string or date to parse.
+     *
+     * @return The parsed date.
+     */
+    date(str: string|Date): Date {
+        return parseDate(str);
+    },
+    /**
+     * Parse a xml document to an object.
+     *
+     * @param str The xml parse.
+     *
+     * @return The parsed object.
+     */
+    xml<T>(str: string): T {
```

Comment:
This should have the return type `unknown` and a function checking for the correct type should be added instead:

```
function isWhatever(arg: unknown): arg is Whatever {
    // Perform validation of type here.
}
```